packages/cli/src/executions/executions.service.ts

```diff
@@ -223,8 +223,19 @@ export class ExecutionsService {
 			});
 		}
 
-		let query = Db.collections.Execution.createQueryBuilder()
-			.select()
+		// Omit `data` from the Execution since it is the largest and not necesary for the list.
+		let query = Db.collections.Execution.createQueryBuilder('execution')
+			.select([
+				'execution.id',
```

Comment:
Could we use [hidden columns](https://typeorm.io/select-query-builder#hidden-columns) to exclude the `data` column everywhere when not explicitly mentioned?

Reply:
I think it might do more harm than good by hiding it. Usually we do want to fetch the `data` from the executions since it's the main content to expect from it. Just when listing is that it's not super important, so I think we're ok this way. Let me know if you think otherwise.